cosmos/airflow/graph.py

```diff
@@ -268,6 +298,17 @@ def create_task_metadata(
         return None
 
 
+def is_detached_test(node: DbtNode) -> bool:
```

Comment:
may be "converging" instead of "detached" here?

just my thought, but okay with detached too.

Reply:
I was checking the meanings in the dictionary:
* converging:  tend to meet at a point.
* detached: separate or disconnected.

I feel detached is closer from the perspective of the test, where we're detaching it from the other model tests and running them "stand-alone".

Since this is an internal implementation detail, we come up with a better name in the future :)